apps/angular/src/app/features/authentification/auth.component.ts

```diff
@@ -0,0 +1,32 @@
+import { ChangeDetectionStrategy, Component, inject } from '@angular/core';
+import { CommonModule } from '@angular/common';
+import { Router, RouterModule } from '@angular/router';
+import { MatCardModule } from '@angular/material/card';
+import { URL_PATHS } from '@js-camp/core/utils/url-paths';
+
+const DEFAULT_TITLE = 'Page';
+
+/** Auth component for styles. */
```

Comment:
Can you explain what "for styles" mean?
```suggestion
/** Auth component. */
```

Reply:
At first this component should be `dumb` with no logic. Just for style purposes. Deleted this comment for now :+1: